lib/V8/v8-utils.cpp

```diff
@@ -712,7 +712,7 @@ auto getEndpoint(v8::Isolate* isolate, std::vector<std::string> const& endpoints
     if (endpoint.find(':') == std::string::npos) {
       endpoint.append(":80");
```

Comment:
shouldn't we also move to port 443 here?

Reply:
https://github.com/arangodb/arangodb/pull/11236#discussion_r397990258